VitruvioHost/Plugins/Vitruvio/Source/Vitruvio/Private/VitruvioModule.cpp

```diff
@@ -99,8 +102,9 @@ class FLoadResolveMapTask
 			const std::wstring AbsoluteRpkPath(TCHAR_TO_WCHAR(*FPaths::ConvertRelativePathToFull(RpkPath)));
 			const std::wstring AbsoluteRpkFolder(TCHAR_TO_WCHAR(*FPaths::Combine(FPaths::GetPath(FPaths::ConvertRelativePathToFull(RpkPath)),
 																				 FPaths::GetBaseFilename(UriPath, true) + TEXT("_Unpacked"))));
-			const std::wstring RpkFileUri = prtu::toFileURI(AbsoluteRpkPath);
 
+			const std::wstring RpkFileUri = prtu::toFileURI(AbsoluteRpkPath);
+			IFileManager::Get().DeleteDirectory(AbsoluteRpkFolder.c_str(), false, true);
```

Comment:
Would it be possible to delete the RPK file after it has been unzipped?

Reply:
Good point! Added with c9febb8